docker-compose.yml

```diff
@@ -1,12 +1,22 @@
 version: "3.7"
 services:
   rabbitmq:
-    image: rabbitmq:3
+    image: rabbitmq:3-alpine
+  influxdb:
+    image: influxdb:1.7-alpine
+    environment:
+      - INFLUXDB_DB=fixm
+      - INFLUXDB_USER=user
+      - INFLUXDB_USER_PASSWORD=password
   sos-journaler:
     image: sync-or-swim/sos-journaler
     environment:
       - RABBITMQ_HOST=rabbitmq
       - RABBITMQ_QUEUE_NAME=fixm
+      - INFLUXDB_HOSTNAME=influxdb
+      - INFLUXDB_DATABASE=fixm
+      - INFLUXDB_USERNAME=user
```

Comment:
difference between INFLUXDB_USER and INFLUXDB_USERNAME?

Reply:
INFLUXDB_USER is used to tell InfluxDB to create a user with the given name. INFLUXDB_USERNAME is used by the journaler to know what user to log into InfluxDB with. Would it be less confusing if I just called ours INFLUXDB_USER?

Reply:
are they always going to be the same value?

Reply:
Yeh

Reply:
Then probably. Why have the user set two variables to do the same thing?